docs/CONTRIBUTING.md

```diff
@@ -54,7 +54,13 @@ To run the tests for a single package, change to package directory and run:
 go test .
 ```
 
-__Note:__ Unit tests must pass for your PR to get merged.
+To run the e2e test for the project:
+
+```
+make test-e2e
+```
+
+__Note:__ Unit and e2e tests must pass for your PR to get merged.
```

Comment:
Should mention that this requires an existing Kubernetes cluster and uses your current cluster context.

Reply:
Mention that you need to build the kind cluster with `./hack/kind-dev-cluster.sh`